src/java.base/share/classes/java/nio/file/Files.java

```diff
@@ -636,8 +637,9 @@ public static Path createFile(Path path, FileAttribute<?>... attrs)
      *          if the array contains an attribute that cannot be set atomically
      *          when creating the directory
      * @throws  FileAlreadyExistsException
-     *          if a directory could not otherwise be created because a file of
-     *          that name already exists <i>(optional specific exception)</i>
+     *          if a directory could not otherwise be created because a file or
```

Comment:
It is pre-existing wording, but I don't know that "otherwise" has any meaning here.
It reads the same whether "otherwise" is present or not.

Reply:
> I don't know that "otherwise" has any meaning here.

No, I also don't think it does.